include/random_walks/gaussian_exact_hmc_walk.hpp

```diff
@@ -0,0 +1,190 @@
+// VolEsti (volume computation and sampling library)
+// Copyright (c) 2021 Vissarion Fisikopoulos
+// Copyright (c) 2021 Apostolos Chalkis
+
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+#ifndef RANDOM_WALKS_GAUSSIAN_EXACT_HMC_WALK_HPP
+#define RANDOM_WALKS_GAUSSIAN_EXACT_HMC_WALK_HPP
+
+#include "sampling/sphere.hpp"
+
+
+
+// Exact HMC for sampling from the spherical Gaussian distribution
+
+struct ExactHMCGaussianWalk
```

Comment:
Naming consistency comments as in #142 .

Reply:
Thanks. Done.